packages/backend/src/index.ts

```diff
@@ -45,11 +45,36 @@ function makeCreateEnv(loadedConfigs: AppConfig[]) {
 
   return (plugin: string): PluginEnvironment => {
     const logger = getRootLogger().child({ type: 'plugin', plugin });
-    const database = knex({
-      client: 'sqlite3',
-      connection: ':memory:',
-      useNullAsDefault: true,
-    });
+    // Supported DBs are sqlite and postgres
+    const isPg = [
+      'POSTGRES_USER',
+      'POSTGRES_HOST',
+      'POSTGRES_PASSWORD',
+    ].every(key => Object.keys(process.env).includes(key));
+
+    let knexConfig;
+
+    if (isPg) {
+      knexConfig = {
+        client: 'pg',
+        useNullAsDefault: true,
+        connection: {
+          port: process.env.POSTGRES_PORT,
+          host: process.env.POSTGRES_HOST,
+          user: process.env.POSTGRES_USER,
+          password: process.env.POSTGRES_PASSWORD,
```

Comment:
Can we grab this from config? 😅

Reply:
Should we? I mean secrets in the config, idk, is this what we decided to do?

Reply:
The config loader has support for secrets from multiple sources, so it's more flexible than this. You can load it from files and whatnot too.